tests/console/consoletest_setup.pm

```diff
@@ -36,6 +36,11 @@ sub run {
     script_run "ip addr show";
     save_screenshot;
 
+    # Stop serial-getty on serial console to avoid serial output pollution with login prompt
+    my $serial_console = get_var('SERIALDEV') ? get_var('SERIALDEV') : 'ttyS0';
```

Comment:
`my $serial_console = get_var('SERIALDEV') || 'ttyS0';`

Reply:
Right, that's better way. Thanks!